package.json

```diff
@@ -41,6 +39,7 @@
     "chrome-devtools-frontend": "1.0.612830",
     "eslint": "^5.9.0",
     "mocha": "^5.2.0",
+    "rimraf": "^2.6.2",
```

Comment:
Do you need this?

Reply:
Yes, it is dev dependency and required for build script.

Reply:
still lgtm